Makefile

```diff
@@ -39,16 +39,16 @@ test_lint:
 	}
 
 test_multi:
+	cd network; \
 	for a in $$( seq 10 ); do \
-	  cd services/identity; \
-	  go test -v -race -p=1 -short ./...; \
+	  go test -v -race -run Stress; \
```

Comment:
Yes, it's a placeholder for any tests you might want to run for a couple of times in Travis. I'll add a comment.